packages/site/src/components/Card.tsx

```diff
@@ -3,9 +3,9 @@ import styled from 'styled-components';
 
 type CardProps = {
   content: {
-    title: string;
-    description: string;
-    button: ReactNode;
+    title?: string;
+    description: string | ReactNode;
```

Comment:
I believe `string` is already a valid `ReactNode`, so this could potentially be:

```suggestion
    description: ReactNode;
```